docs/README.md

```diff
@@ -297,20 +297,20 @@ services:
   nginx-proxy:
     image: nginxproxy/nginx-proxy
     ports:
-      - "80:80"
+      - 80:80
     volumes:
       - /var/run/docker.sock:/tmp/docker.sock:ro
     environment:
       HTTPS_METHOD: nohttps
   myapp:
     image: jwilder/whoami
     expose:
-      - "8000"
+      - 8000
     environment:
       VIRTUAL_HOST: myapp.example
-      VIRTUAL_PORT: "8000"
+      VIRTUAL_PORT: 8000
     labels:
-      com.github.nginx-proxy.nginx-proxy.loadbalance: "hash $$remote_addr;"
+      com.github.nginx-proxy.nginx-proxy.loadbalance: hash $$remote_addr
```

Comment:
This is invalid. The `;` needs to be in the string.

Reply:
My apologies, accidental removal. 
Resolved, will be pushed in the next commit